proto/stride/stakeibc/trade_route.proto

```diff
@@ -49,7 +49,7 @@ message TradeRoute {
     (gogoproto.nullable) = false
   ];
   // timestamp that the price was last updated
-  google.protobuf.Timestamp last_price_update_time = 11
+  google.protobuf.Timestamp price_update_time = 11
```

Comment:
this is fine for now, but in the future I prefer using the word "timestamp" in the name when the type is a timestamp because there are so many possible time formats -- human readable, a difference -> duration, unix time in seconds, timestamp in nanos, etc. and in different places across the codebase we use time meaning any of these.

Reply:
the protobuf type is Timestamp, but the go type is time.Time